TechnicalTask-lvl1/Managers/Network/UserNetworkManager.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+import Combine
+
+protocol APIClientProtocol {
+    func response<T: Codable>(_ controller: ApiController,
+                              receiveOnQueue: OperationQueue) -> AnyPublisher<UploadResponse<T>, NetworkError>
+}
+
+enum UploadResponse<T: Decodable> {
+    case response(data: T)
+}
+
+final class UserNetworkManager: APIClientProtocol {
+    enum HTTPMethod: String {
+        case GET
+    }
+
+    func response<DataType: Codable>(_ controller: ApiController,
```

Comment:
Just out of curiosity. Why is the method called `response(...`  and not `request(...`?